README.md

```diff
@@ -1,2 +1,2 @@
-# swift-mongoc
-Swift bindings for libmongoc
+# End of Life Notice
+The MongoDB library of Swift bindings for libmongoc (swift-mongoc) is now officially end-of-life (EOL). No further development, bugfixes, enhancements, documentation changes or maintenance will be provided by this project and pull requests will no longer be accepted.
```

Comment:
Add a sentence directing people to https://github.com/mongodb/mongo-swift-driver for developing server side Swift applications with MongoDB.

Reply:
Added!